mappings/net/minecraft/client/render/BlockBreakingProgress.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS net/minecraft/class_3191 net/minecraft/client/render/PartiallyBrokenBlockEntry
+CLASS net/minecraft/class_3191 net/minecraft/client/render/BlockBreakingProgress
```

Comment:
`BreakingBlock`/`CrumblingBlock`?

Reply:
This isn't a block but info on block breaking, so i call it a breaking progress

Reply:
I'd call this `BreakingBlockInfo`, since it's more than just a progress, it also has the position of the block.